app/common/global/event-handlers.js

```diff
@@ -20,4 +22,11 @@ function (
     $rootScope.toggleModalVisible = function (state) {
         $rootScope.modalVisible = (typeof state !== 'undefined') ? state : !$rootScope.modalVisible;
     };
+
+    // Clear filters when navigating to home
+    $rootScope.$on('$locationChangeSuccess', function () {
```

Comment:
I don't think this should happen here. I'd like to minimize global events over time.. since they make it less obvious where this happens.

Ideally this should happen in the post-views.controller.js instead, however calling `clearFilters()` will clear the filters when we switch map/timeline views. Maybe we can check the previous location there? otherwise we need to look at tracking state (are we on home, collection or saved searches?) in PostFilters..